src/amo/pages/UserProfileEdit/index.js

```diff
@@ -420,8 +420,8 @@ export class UserProfileEditBase extends React.Component<Props, State> {
       : i18n.sprintf(
           i18n.gettext(`Tell users a bit more information about this user.
             These fields are optional, but they'll help other users get to know
-            %(username)s better.`),
-          { username: user.name },
+            %(userName)s better.`),
```

Comment:
Nit: `username` seems fine to me. I think we often use that, rather than `userName`, so I'm not sure the changes in this file are necessary.

Reply:
I changed those occurrences because we now use `user.name` and not `user.username` anymore. I've seen `userName` in the review code and I liked it.